ecosystem/sep-0018.md

```diff
@@ -50,6 +50,23 @@ wallet.eth
 * **Term REGEXP:** `[_a-z][_a-z0-9]*`
 * **Key REGEXP:** `[_a-z][_a-z0-9]*(\.[_a-z][_a-z0-9]*)*`
 
+## Namespace Registry
+
+To guarantee interoperability and avoid naming conflicts, vendor-specific data
+must live under their reserved namespace:
+
+- `app`: Space dedicated to vendor-specific keys. Keys must be defined under
```

Comment:
Let's refer to this as app specific rather than vendor specific since the field is named app it means introducing less concepts and vocabulary that the reader must understand. It's a small factor.
```suggestion
To guarantee interoperability and avoid naming conflicts, app-specific data
must live under their reserved namespace:

- `app`: Space dedicated to app-specific keys. Keys must be defined under
```